packages/app-builder/src/components/ui/sidebar-item/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { styled } from '@linaria/react'
+
+export const SidebarItemDiv = styled.div<{ visible?: boolean; height?: string }>`
+  height: ${(props) => {
+    return props.visible && props.height && props.height !== 'full' ? `${props.height}` : 'auto'
```

Comment:
display: none?

Reply:
think this is just badly named, renamed to "expanded"